descriptors/analysis_engines/demo.xml

```diff
@@ -174,14 +174,6 @@
     <flowConstraints>
       <fixedFlow>
        <node>CollectionReader</node>
-       <node>ImagePreprocessor</node>
-       <node>PointCloudFilter</node>
-       <node>NormalEstimator</node>
-       <node>PlaneAnnotator</node>
-       <node>ImageSegmentationAnnotator</node>
-       <node>PointCloudClusterExtractor</node>
-       <node>ClusterMerger</node>
-       <node>Cluster3DGeometryAnnotator</node>	
        <node>StorageWriter</node>
```

Comment:
these changes break the tutorials online; consider using the same AE definitions from the other uTest;